src/components/StaticQAndA.tsx

```diff
@@ -0,0 +1,76 @@
+import styles from './StaticQAndA.module.scss';
+import Markdown from './Markdown';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faSquareCheck,
+  faCircleCheck,
+} from '@fortawesome/free-solid-svg-icons';
+import { faSquare, faCircle } from '@fortawesome/free-regular-svg-icons';
+
+import { useTranslation } from 'react-i18next';
+import { AnswerType, QuestionTypes, QuestionType } from '../types';
+
+const StaticQAndA = ({
+  question,
+  highlightedAnswers,
+  collapsed,
+}: {
+  question: QuestionType;
+  highlightedAnswers: AnswerType[];
+  collapsed: boolean;
+}) => {
+  const { t } = useTranslation();
+
+  const wasSelected = (answer: AnswerType) =>
+    highlightedAnswers.some((sel) => sel.id === answer.id);
+
+  const indicatorClass = (answer: AnswerType) =>
+    answer.correct ? styles.correct : styles.wrong;
+
+  const selectedClass = (answer: AnswerType) =>
+    wasSelected(answer) ? styles.selected : '';
+
+  const correctHint = (answer: AnswerType) =>
+    answer.correct ? t('qAndA.correct') : t('qAndA.incorrect');
+
+  const icon = (answer: AnswerType) => {
+    const type =
+      question.type === QuestionTypes.SingleChoice ? 'single' : 'multiple';
+    if (type === 'single') {
+      return wasSelected(answer) ? faCircleCheck : faCircle;
+    } else {
+      return wasSelected(answer) ? faSquareCheck : faSquare;
+    }
+  };
+
+  return (
+    <div className={collapsed ? styles.collapsed : ''}>
```

Comment:
I decided not to use `<article>` as this implies that the content is a standalone piece of content, which is not exactly the case within the context of a table. This is part of a cell and it it semantically connected to the other parts of the table.